packages/main/src/themes/base/Panel-parameters.css

```diff
@@ -17,4 +17,8 @@
 .ui5-content-density-compact,
 .sapUiSizeCompact {
 	--_ui5_panel_header_button_wrapper_padding: .5625rem .375rem;
+}
+
+.ui5-panel-heading-wrapper {
+	--_ui5_panel_header_background_color: var(--sapGroup_TitleBackground);
```

Comment:
Isn't this applicable only for horizon? As far as I can see both belize and fiori_3 use --sapBackgroundColor. Pleae check with Niya what is the better approach:
- to have here --sapBackgroundColor and change the parameter value for horizon themes
- to have it in the way you implemented it

Reply:
You're right. I've checked with Niya and I will change the base one and set the horizon :)